admin_manual/configuration_files/files_access_control.rst

```diff
@@ -49,6 +49,20 @@ This example blocks access to any folder with the tag ``Confidential``.
     .. figure:: images/files_access_control_collaborative_tags.png
        :alt: Deny access based on collaborative tag
 
+Prevent uploading of specific files
+-------------------------
```

Comment:
I can't see the page online as the search engine doesn't work...
https://docs.nextcloud.com/server/10/admin_manual/search.html?q=File+access+control#